fecgov/fec-cms
# Office groups to for updates pages

## Summary
Adds the  "author groups" field the the Author model. One can now designate an author group for authors they want to show up in an inclusive list when they choose that  group as an author for an updates page.

A templatetag named "author_groups.py" has been added to the /Updates pages author-list.html template to conditionally include author groups where applicable

## Impacted areas of the application
modified:   fec/constants.py
	new-file:    fec/templates/partials/author-groups.html
	modified:   fec/templates/partials/author-list.html
	new file:    home/migrations/0104_auto_20181202_1431.py
	modified:   home/models.py
	new file:   home/templatetags/author_groups.py

## How to test

  This might include:
-  checkout and run this branch locally: feature/author-groups-for-updates-pages
- ./manage.py migrate
- go to http://127.0.0.1:8000/admin/home/author/
- Create an author named 'Press Office' or 'Information Division' (don't worry about email, bio fields etc)
 - Edit one of the existing other authors.  Under Author groups at the bottom, choose "Press authors" or "Information Division authors"
- Do one of the following:
   - Edit an exiting press release, weekly digest, or record page
       - Change the first authors name to the group you added the author to, (either Press Office  or Information Division)
    - Create a new  press release, weekly digest, or record page
       - Add an author and choose ,as that author , the group you added the author too, (either Press Office  or Information Division)
- pPeview or publish the page to see that all authors added to this group will show up when this gorup is chosen as the Author for the page. 

____
